tests/conftest.py

```diff
@@ -1390,13 +1389,24 @@ def collect_db_dump_on_duts(request, duthosts):
             db_id = db_config['DATABASES'][db]['id']
             dbs.add(db_id)
 
-        # Collect DB dump
-        duthosts.file(path = db_dump_path, state="directory")
-        for i in dbs:
-            duthosts.shell("redis-dump -d {} -y -o {}/{}".format(i, db_dump_path, i))
+        namespace_list = duthosts[0].get_asic_namespace_list() if duthosts[0].is_multi_asic else ['']
+        if namespace_list != ['']:
```

Comment:
The above 2 lines can be more pythonic like below.
```
        namespace_list = duthosts[0].get_asic_namespace_list() if duthosts[0].is_multi_asic else []
        if namespace_list:
```

Reply:
OK